host/driver_offline/include/device_convolution_backward_weight_implicit_gemm_v4r4r2_xdlops_atomic_nchw_kcyx_nkhw.hpp

```diff
@@ -95,11 +95,9 @@ void device_convolution_backward_weight_implicit_gemm_v4r4r2_xdlops_atomic_nchw_
     const auto GemmN      = Y * X * C;
     const auto GemmKTotal = N * Ho * Wo;
 
-    const auto GemmK = GemmKTotal / GemmK1;
-
     const auto GridMN        = GemmM * GemmN / (GemmMPerBlock * GemmNPerBlock);
     const index_t GemmKBatch = std::max(desired_grid_size / GridMN, 1);
-    const index_t BatchLen   = std::ceil(GemmK * 1.0 / (GemmKPerBlock * GemmKBatch));
+    const index_t BatchLen   = std::ceil(GemmKTotal * 1.0 / (GemmK1 * GemmKPerBlock * GemmKBatch));
```

Comment:
No need to calculate BatchLen, right?

Could you remove it

Reply:
done